### PR TITLE
Add maxdistance parameter to chrony

### PR DESCRIPTION
**Pull request by davidphay**
This pull request allow users to edit `maxdistance` param in chrony config.

It usefull to fix the issue when the distance with the source is greater than the default value (3)

### CODE REVIEW

**Comment by davidphay** [comment]
@geerlingguy is there something I can do to get this PR merged?

File: defaults/main.yml [hunk 1]
```diff
@@ -25,3 +25,5 @@ ntp_restrict:
 ntp_cron_handler_enabled: false
 
 ntp_tinker_panic: false
+
+maxdistance: '3'
```
Comment (geerlingguy) [inline defaults/main.yml:29]: This should probably be something like `ntp_chrony_maxdistance`, and it would need to be documented in the README.

**Review by geerlingguy — CHANGES_REQUESTED** [review]

**Comment by davidphay** [comment]
@geerlingguy  It should be fine now.

I update the readme & the param name

**Comment by davidphay** [comment]
@geerlingguy I fixed the yaml lint syntax!

**Comment by davidphay** [comment]
@geerlingguy What's the issue ?

**Comment by geerlingguy** [comment]
Looks like build failures due to the docker python library

File: README.md [hunk 2]
```diff
@@ -66,6 +66,12 @@ Whether to restart the cron daemon after the timezone has changed.
 
 Enable tinker panic, which is useful when running NTP in a VM.
 
+    ntp_tinker_panic: true
```
Comment (geerlingguy) [inline README.md:69]: The ordering of these descriptions and examples is off, the `ntp_chrony_maxdistance` should be up here.

**Review by geerlingguy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (davidphay) [reply README.md]: Can you check now, it shoudl be good, Thanks !

**Review by geerlingguy — APPROVED** [review]

**Pull request merged**